examples/run_glue_tpu.py

```diff
@@ -0,0 +1,617 @@
+# coding=utf-8
+# Copyright 2019 The Google AI Language Team Authors and The HuggingFace Inc. team.
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Finetuning the library models for sequence classification on GLUE (Bert, DistilBert, XLNet, RoBERTa)."""
+
+from __future__ import absolute_import, division, print_function
+
+import argparse
+import glob
+import logging
+import os
+import random
+
+import numpy as np
+import torch
+import torch_xla.core.xla_model as xm
```

Comment:
I don't like this naming for non-standard modules. Can you refer to these by their full names.

Reply:
In all our examples we use `xm`, `xmp`, `met`, `pl` consistently so I'd rather keep this way for the sake of consistency. I similarly do see `np` throughout Huggingface, but if you feel strongly, I'm happy to call them by full name instead.

Reply:
I see. If that's the XLA standard I guess it's okay. (I feel like numpy is a special case).